### PR TITLE
config: update operator payloads for v0.8.0 release

### PR DESCRIPTION
**Pull request by portersrc**
This covers step 13 from the release checklist [here](https://github.com/confidential-containers/confidential-containers/issues/145)

### CODE REVIEW

File: config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml
Comment (fidencio) [inline config/samples/enclave-cc/base/ccruntime-enclave-cc.yaml:12]: For this one ...

File: config/samples/enclave-cc/sim/kustomization.yaml
Comment (fidencio) [inline config/samples/enclave-cc/sim/kustomization.yaml:8]: ... and this one, could you use ed979ef2952d8b20c2f0abee3d32a3617d2c57bb tag?

enclave-cc merged a bump on Occlum Today, and we want to test with that image.

**Review by fidencio — CHANGES_REQUESTED** [review]
@portersrc, please, just take a look at the comments related to enclave-cc.

**Comment by portersrc** [comment]
> @portersrc, please, just take a look at the comments related to enclave-cc.

I bumped it; it looks like that enclave-cc bundle is still building as of this writing though

**Review by fidencio — APPROVED** [review]
lgtm, assuming the tests will pass.

Thanks, @portersrc!

File: tests/e2e/operator.sh
Comment (wainersm) [inline tests/e2e/operator.sh:69]: Thanks for the fix, @portersrc !

**Review by fitzthum — COMMENTED** [review]
LGTM

**Review by fitzthum — APPROVED** [review]

**Review by stevenhorsman — APPROVED** [review]
LGTM. Thanks!

**Comment by fidencio** [comment]
> @portersrc any special reason to not pin the s390x payload? -> https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml

Oh, that's something that was overlook, I guess, nice catch!

**Comment by wainersm** [comment]
> > @portersrc any special reason to not pin the s390x payload? -> https://github.com/confidential-containers/operator/blob/main/config/samples/ccruntime/s390x/kustomization.yaml
> 
> Oh, that's something that was overlook, I guess, nice catch!

Also we are using the CI payload instead of `quay.io/confidential-containers/runtime-payload`

**Comment by wainersm** [comment]
@portersrc you also need to bump the hash of pre-reqs for peer-pods too

Good opportunity to advertise this: https://github.com/confidential-containers/operator/pull/205 (perhaps someone "cof-cof Chris cof-cof" would like to take over that script).

**Comment by fitzthum** [comment]
> Also we are using the CI payload instead of quay.io/confidential-containers/runtime-payload

I think the idea is to switch to the runtime-payload bundle in step 19 after we add the release tags for kata and enclave-cc. In the past we have sometimes combined this step with that one, though, and we could do that again if people think it would save time.

**Comment by wainersm** [comment]
> > Also we are using the CI payload instead of quay.io/confidential-containers/runtime-payload
> 
> I think the idea is to switch to the runtime-payload bundle in step 19 after we add the release tags for kata and enclave-cc. In the past we have sometimes combined this step with that one, though, and we could do that again if people think it would save time.

Good point @fitzthum , I forgot kata isn't tagged yet. IMHO, anything to speed up **this** release is welcomed, so I am okay to switch the payload on this PR to run the CI jobs one more time

**Comment by portersrc** [comment]
Thanks to @wainersm for catching these two things: pinning s390x runtime-payload-ci and also pinning peer-pods' reqs-payload.
These are incorporated into the latest push.

**Comment by fitzthum** [comment]
@wainersm my hunch for now is to do another PR for step 19 just to make sure we don't confuse ourselves. I don't think that will take too much longer, but I leave it to @portersrc to decide.

**Review by wainersm — APPROVED** [review]
@portersrc now it looks okay to me. Thanks!

**Comment by fitzthum** [comment]
Let's give these test one more chance to behave

**Pull request merged**
